lte/protos/policydb.proto

```diff
@@ -19,6 +19,10 @@ import "lte/protos/mobilityd.proto";
 package magma.lte;
 option go_package = "magma/lte/cloud/go/protos";
 
+message HeaderEnrichment{
+  repeated string URLs = 1;
```

Comment:
I don't think we usually capitalize these, maybe `repeated string urls = 1;` would be more consistent with other proto messages?